src/media/inner.rs

```diff
@@ -1128,7 +1128,9 @@ impl MediaInner {
         let state = QueueState {
             mid: self.mid,
             is_audio: self.kind.is_audio(),
-            use_for_padding: self.kind.is_video() && self.has_tx_rtx(),
+            use_for_padding: self.kind.is_video()
+                && self.has_tx_rtx()
+                && snapshot.last_emitted.is_some(),
```

Comment:
What is the semantic meaning of this third argument?

Reply:
It's saying that we only use queues for padding if we have previously sent something on them, definitionally that thing will be media

Reply:
Could you maybe make that clearer by having `snapshot.has_sent_something()` or similar?

Reply:
Sure